storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -78,6 +78,11 @@ De Nederlandse taal kent veel lange woorden, zoals 'gemeenteraadsverkiezingen'.
 
 Voor verdere informatie: [Wrapping and breaking text op MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_text/Wrapping_breaking_text)
 
+## Andere talen
+
+[Geef programmatisch aan in welke taal tekst in een afwijkende taal is](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts.html).
```

Comment:
This it good advice but it’s not really a matter of typography.

If we want to leave it in until we have a section on internationalisation, we should add an HTML example to make this guideline practical without requiring people to assess the entire WCAG article.

For the same reason, we could add ‘Meer informatie’ or ‘Verder lezen’ or ‘Meer weten?’ to this links (and all others like it, some already do, let’s reuse one phrase). I’d like our documentation to suffice for the reader that understands and accepts our guidelines.